src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -577,6 +577,34 @@ private void testEditInstructorAction() throws Exception {
                 sectionToCheck, sessionToCheck, InstructorCourseEditPage.SESSION_MODIFY_RESPONSES));
 
         courseEditPage.selectRoleForInstructor(editInstructorIndex, "Co-owner");
+
+        ______TS("Verify that one instructor must be visible to students");
+        courseEditPage.makeOtherInstructorsInvisible(8, 8);
```

Comment:
Editing 8 instructors twice is very expensive, especially when done against the live server. Can we use a course with fewer instructors?